packages/expo/src/modules/home/Home.tsx

```diff
@@ -1,23 +1,246 @@
-import React from 'react';
-import {Button, AsyncStorage} from 'react-native';
+import React, {useState, useEffect, useRef, useCallback} from 'react';
+import {StyleSheet, AsyncStorage, Dimensions, ImageSourcePropType, StyleProp} from 'react-native';
+import Animated from 'react-native-reanimated';
+import {PanGestureHandler, State} from 'react-native-gesture-handler';
 import {px2ddp, useTheme} from '@animavita/theme';
 import {Background} from '@animavita/ui/layout';
-import {TabBar} from '@animavita/ui/core';
+import {TabBar, Card} from '@animavita/ui/core';
 import styled from 'styled-components/native';
 
+import Button from '../../../../ui/core/Button/Button';
+import {Profile} from '../../../../ui/@types/profile';
+
+const {
+  greaterThan,
+  lessThan,
+  and,
+  Clock,
+  clockRunning,
+  startClock,
+  stopClock,
+  spring,
+  event,
+  Value,
+  concat,
+  interpolate,
+  Extrapolate,
+  set,
+  cond,
+  eq,
+  neq,
+  call,
+} = Animated;
+
+// TS types of data object to feed the Home screen
+interface Iterators {
+  lastPets: Profile;
+  rest: Profile[];
+}
+
+//Array data to feed the animation
+const Pets: Profile[] = [
+  {
+    id: 1,
+    name: 'Thor',
+    age: 2,
+    image: require('../../../../ui/assets/images/examples/dog1.jpg'),
+  },
+  {
+    id: 2,
+    name: 'Bob',
+    age: 2,
+    image: require('../../../../ui/assets/images/examples/dog2.jpg'),
+  },
+  {
+    id: 3,
+    name: 'Tim',
+    age: 2,
+    image: require('../../../../ui/assets/images/examples/dog3.jpg'),
+  },
+  {
+    id: 4,
+    name: 'Robb',
+    age: 2,
+    image: require('../../../../ui/assets/images/examples/dog4.jpg'),
+  },
+];
+
+//Mathematical equations to calculate the trigonometric coefficient of rotation
+const {width, height} = Dimensions.get('window');
+const rotatedWidth = width * Math.sin((75 * Math.PI) / 180) + height * Math.sin((15 * Math.PI) / 180);
+
+//Animation function to spring animation
+function runSpring(
```

Comment:
What about moving all the animation setup to another file?

Reply:
I liked it, where would you suggest put the file with the config?

Reply:
I think under this directory should be fine.